app/views/pages/main.html.haml

```diff
@@ -3,14 +3,14 @@
 
 %h2.ui.header transientBug
 %p
-  transientBug offers basic bookmark functionality, akin to pocket, pinboard, delicious and others, but with a few benifits:
+  transientBug offers basic bookmark functionality, akin to pocket, pinboard, delicious and others, but with a few benefits:
   %ul.list
     %li
       %b Cheap
-      If it works out well I might look into some low cost plans, but for now its free.
+      If it works out well I might look into some low cost plans, but for now its free. That being said, transientBug is currently in a closed, invite only registration right now, until it gets to a stable and complete feature-set that is ready for public use.
     %li
-      %b Simple
-      It should be featurefull and advanced where applicable, but the interface should try to remain as simple as I can make it (This is a work in progress).
+      %b Effective, Efficient but Advanced
+      I really want something that has one-click & painless bookmarking, but it should also be feature-full and advanced enough to not be just a toy.
```

Comment:
> Effective, Efficient but Advanced
> I really want something that has one-click & painless bookmarking, but it should also be feature-full and advanced enough to not be just a toy.

- ~but Advanced~ `capable`
- `but it should also be feature-full and advanced enough to not be just a toy` how about `and also has more advanced features like caching, searching, and works across multiple devices.`